<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsPage4</class>
 <widget class="QWidget" name="SettingsPage4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Automatic formatting</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="trimWhiteSpace">
        <property name="toolTip">
         <string>Removes spaces before first and after last word</string>
        </property>
        <property name="text">
         <string>Trim whitespace</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="safeFileNames">
        <property name="toolTip">
         <string>Removes illegal characters from filenames. Useful for portable media players etc.</string>
        </property>
        <property name="text">
         <string>Use FAT-compatible filenames</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="convertExtensions">
        <property name="toolTip">
         <string>Convert filename extensions to either lower-case or upper-case</string>
        </property>
        <property name="text">
         <string>Convert filename extensions to</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QRadioButton" name="extensionsLowerCase">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Lower-case</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="extensionsUpperCase">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Upper-case</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="removeDiscFromAlbum">
        <property name="text">
         <string>Remove disc number from album name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Searches for &quot;(Disc %d)&quot; and &quot;(CD %d)&quot;</string>
        </property>
        <property name="indent">
         <number>35</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>convertExtensions</sender>
   <signal>toggled(bool)</signal>
   <receiver>extensionsLowerCase</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>convertExtensions</sender>
   <signal>toggled(bool)</signal>
   <receiver>extensionsUpperCase</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
